docs/pages/concept/web.md

```diff
@@ -0,0 +1,139 @@
+# Web Renderer
+
+## Overview
+
+Web rendering is an experimental feature that lets you render websites.
+Furthermore, you can place other components on the website. We refer to this process as embedding.
+
+Make sure you have a compositor version built with web renderer support. The web renderer introduces additional dependencies and significantly increases the size of the compositor binaries. To minimize that impact, we are supporting two versions of the compositor, one with web renderer support and one without it.
+
+:::tip
+You can view a working example [here](https://github.com/membraneframework/video_compositor/blob/master/examples/web_view.rs)
+:::
+
+## Embedding components
+
+Embedding is a process of displaying child components on a website. You can define the child components in the `children` field of the web view.
+The child component IDs have to correspond to the IDs of HTML elements.
+The web renderer embeds the children's frames in the specified HTML elements.
+
+### Embedding methods
+
+There are 3 embedding methods available:
+
+- `chromium_embedding` - Frames produced by child components are passed directly to a chromium instance and displayed on an HTML canvas. Passing frames to the chromium instance introduces one more copy operation on each input frame, which may cause performance problems for a large number of inputs. The HTML elements used for embedding have to be canvases.
+- `native_embedding_over_content` - Renders frames produced by child components on top of the website's content.
+- `native_embedding_under_content` - Renders frames produced by child components below the website's content. The website needs to have a transparent background. Otherwise, it will cover the frames underneath it.
+
+`native_embedding_over_content` is the default embedding method.
+You can change it in the [register renderer request](../api/routes#register-renderer). For example:
```

Comment:
This is just a personal opinion, so don't change that just on account of that, but I have seen that in PRs of others and wanted to raise that point.

I often see markdown being written where each sentence is written in a new line. After rendering on a website this will become a single block of text. I prefer to write text like that in a single line and rely on text editor wrapping or to add new lines manually when reaching a specific width, not at the end of the sentence.

The main problem with this current approach is that you don't see how text will be formatted on the website, 
newlines here give an illusion of formatting. For example, if you write 10 sentences like that, each from newline, it will probably look fine in markdown, but on the website, it will be rendered as a huge chunk of text.

cc @WojciechBarczynski @jerzywilczek any opinions on that?

Reply:
We can ban this if you want, I don't feel strong either way

Reply:
We can ban this if you want